tests/unit/models/file-info-test.js

```diff
@@ -96,46 +96,52 @@ describe('Unit - FileInfo', function(){
     });
   });
 
-  it('renders a menu with an overwrite option', function(){
+  it('renders a menu with an overwrite option', function(done){
     var fileInfo = new FileInfo(validOptions);
 
     ui.waitForPrompt().then(function(){
       ui.inputStream.write('Y' + EOL);
     });
 
-    return fileInfo.confirmOverwrite().then(function(action){
+    fileInfo.confirmOverwrite().then(function(action){
       var output = ui.output.trim().split(EOL);
       expect(output.shift()).to.match(/Overwrite.*\?/);
       expect(action).to.equal('overwrite');
     });
+
+    return done();
```

Comment:
same as above, this `completes` before the entire `confirmOverwrite` part of the test.